.circleci/config.yml

```diff
@@ -20,10 +20,15 @@ jobs:
             docker-compose exec web composer lint
             docker-compose exec assets npm run lint
 
+      - run:
+          name: Build assets
+          command: |
+            docker-compose exec assets npm run assets
+            git status
+            git status | grep "nothing to commit, working tree clean"
```

Comment:
just curious, why do we need to run `git status` twice?

Reply:
It's so when the 2nd one fails we can see above what files it's failing on. Not strictly required, but useful when trying to debug failures.

Reply:
Ah, clever.

Reply:
The current failure is a case in point: https://circleci.com/gh/wikimedia/svgtranslate/140